### ISSUE TITLE
Content pages: H1 Publications tag needs to come before the content filters but after the main body.

### ISSUE THREAD

**Issue by MaysaB** [zooniverse/front-end-monorepo#3584]
## Package
app-content-pages

## Is your feature request related to a problem? Please describe.
#1699 - flagged when resolving this issue.

When using a screen reader, the content filters come before the H1 publications tag, making it difficult to understand what the page is actually for. The 'h1' tag is within the main body but needs to be separate so that the publications heading can be read before the sidebar filters without the content of the main-body coming before the sidebar.

## Describe the solution you'd like
Separate the h1 tag and description from the main body. Put it before the sidebar content filter.

## Describe alternatives you've considered
Label to help navigate screen reader to the h1 tag.

## Additional context
It may mean that TwoColumnLayout will have to be altered to become a three column layout, or the H1 tag added to the AboutHeader, maybe.

**Comment by eatyourgreens** [comment]
I've been thinking about the Publications page and another possible enhancement would be to make the list in the sidebar a list of navigation links, rather than buttons.

Eg. this link in the sidebar
```html
<a href="#space">Space</a>
```
linked to this heading in the main content
```html
<h2 id="space">Space</h2>
```
would make the content easier to navigate in a screen reader and would give each category a direct link, which could be bookmarked eg. https://www.zooniverse.org/about/publications#space.

This would also reduce the amount of JavaScript that's needed to navigate the page, or provide a fallback when JS isn't available.

**Comment by eatyourgreens** [comment]
Also, the list in the sidebar should be marked up as a list (`<ul>`) rather than using `div` throughout the markup. Lists are easier to navigate in a screen reader (#667.)
